### ISSUE TITLE
Check for z-buffer presence on first render / during initialization.

### ISSUE THREAD

**Issue by gw3583** [servo/webrender#3188]
We can probably just query the depth bits in the bound framebuffer, or similar.

See https://github.com/servo/webrender/issues/3186 for context.

**Comment by ChinYing-Li** [comment]
Hi, I would love to help if this issue is still valid!

**Comment by kvark** [comment]
I don't think this is relevant. WebRender issue tracker has been effectively moved to BugZilla: https://bugzilla.mozilla.org/buglist.cgi?product=Core&component=Graphics%3A%20WebRender&resolution=---
